src/base/bittorrent/session.h

```diff
@@ -144,6 +144,14 @@ namespace BitTorrent
         };
         Q_ENUM_NS(ResumeDataStorageType)
 
+        enum class DiskIOType : int
```

Comment:
The enum classes should be sorted alphabetically. So this should be moved below `Q_ENUM_NS(ChokingAlgorithm)`.